frontend/src/scenes/insights/InsightTooltip/InsightTooltip.scss

```diff
@@ -56,7 +60,7 @@
                 }
             }
         }
-        .LemonTable__content > table > tbody {
+        .LemonTable__content > table {
```

Comment:
This is on purpose. We were only targetting the body but the same things in the header apply as in the body